presto-hive/src/test/java/com/facebook/presto/hive/TestCteExecution.java

```diff
@@ -1261,9 +1263,15 @@ protected Session getSession()
     protected Session getMaterializedSession()
     {
         return Session.builder(super.getSession())
+                .setSystemProperty(VERBOSE_OPTIMIZER_INFO_ENABLED, "true")
```

Comment:
Do we want to include this in the test?  Won't it increase the log size substantially?

Reply:
We need this for confirming that the cte was materialized. This doens't increase logging that much since most of the info is added to the session when this property is on

Reply:
Got it, I had forgotten how this property works.